DOCS/man/options.rst

```diff
@@ -5700,6 +5700,32 @@ them.
     results, as can missing or incorrect display FPS information (see
     ``--display-fps-override``).
 
+``--egl-config-id=<ID>``
+    (EGL only)
+    Select EGLConfig with specific EGL_CONFIG_ID.
+    Rendering surfaces and contexts will be created using this EGLConfig.
+    You can use ``--msg-level=vo=trace`` to obtain a list of available configs.
+
+``--egl-output-format=<auto|rgb8|rgb10|rgba8|rgb10_a2|rgb16f|rgba16f>``
+    (EGL only)
+    Select a specific EGL output format to utilize for OpenGL rendering.
+    This option is mutually exclusive with ``--egl-config-id``.
+    "auto" is the default, which will pick the first usable config
+    based on the order given by the driver.
+
+
+    .. note::
+
+        Given there is no reliable API to query desktop bit depth in EGL,
+        it is recommended to manually set this option
+        according to the bit depth of your display.
+        This option also affects the auto-detection of ``--dither-depth``.
```

Comment:
is every user really now supposed to go and set this in his config?
and that has a perceivable advantage?

Reply:
even if that is the case this option would be placebo for 90% of users since mpv requests at least 8 bits anyway, so I'd formulate the docs different than "everyone should set this"

Reply:
modified